### PR TITLE
rails 6 deprecation of case-sensitive uniqueness

### PR DESCRIPTION
**Pull request by akostadinov**
follow-up on #3507

What really bothers me is that I can't see deprecation error locally. Although `config.active_support.deprecation` is configured.

**update:** I figured out. In prod database we use `utf8_general_ci` while locally we have `utf8_bin`. But the deprecation message from `/home/user/.asdf/installs/ruby/2.6.7/lib/ruby/gems/2.6.0/gems/activerecord-6.0.6.1/lib/active_record/connection_adapters/abstract_mysql_adapter.rb` only applies when column collation is case insensitive. That's why locally it didn't trigger. If locally the `ci` collation is being used, the deprecation message is shown up.

### CODE REVIEW

**Comment by mayorova** [comment]
Re: why it was not seen locally, https://github.com/3scale/porta/pull/3507 description also mentioned this :wink::
>This is only observed on SaaS database, because its tables use case-insensitive collation. In on-premises (or locally) the warning is not shown, because there is no mismatch between the table collation and the validator behavior.

This was driving me mad also, until I figured out :grimacing:

**Comment by akostadinov** [comment]
> Re: why it was not seen locally, #3507 description also mentioned this 😉:

tl;dr; :man_shrugging:
